gax-java/gax-httpjson/src/main/java/com/google/api/gax/httpjson/longrunning/stub/HttpJsonOperationsStub.java

```diff
@@ -223,21 +230,70 @@ public static final HttpJsonOperationsStub create(
       TypeRegistry typeRegistry)
       throws IOException {
     return new HttpJsonOperationsStub(
-        OperationsStubSettings.newBuilder().build(), clientContext, callableFactory, typeRegistry);
+        OperationsStubSettings.newBuilder().build(),
+        clientContext,
+        callableFactory,
+        typeRegistry,
+        new HashMap<>());
+  }
+
+  public static final HttpJsonOperationsStub create(
+      OperationsStubSettings settings, Map<String, String> customOperationHttpBindings)
```

Comment:
This approach is fine but I would try to explore the possibility of add the `customOperationHttpBindings` to `settings` or `clientContext`. It would be cleaner and we don't have to have so many overloaded methods.

Reply:
(From what I understand) The core of the logic which we can change in the generated library comes from:
```
  protected HttpJsonSpeechStub(
      SpeechStubSettings settings,
      ClientContext clientContext,
      HttpJsonStubCallableFactory callableFactory)
      throws IOException {
    this.callableFactory = callableFactory;
    this.httpJsonOperationsStub =
        HttpJsonOperationsStub.create(
            clientContext,
            callableFactory,
            typeRegistry,
...
```
at https://github.com/googleapis/google-cloud-java/blob/9c0ddf2879b0e3ddf29b5d67cd44f0336f492134/java-speech/google-cloud-speech/src/main/java/com/google/cloud/speech/v1/stub/HttpJsonSpeechStub.java#L181-L188

The `SpeechStubSettings settings` parameter is different from `OperationsStubSettings` which the `HttpJsonOperationsStub.create()` is expecting. The other approach I'm looking at is to try and add it either the ClientContext (currently leaning towards not putting it here) or the `HttpJsonStubCallableFactory` (Need to do a bit more digging into this).

The overloaded methods were the easiest way I could get the CI builds to pass. Ideally, I would just have each of the `.create()` methods take in another `Map` parameter, but I'm having issue with the build passing. I can show you this tomorrow.